server/subsonic/helpers.go

```diff
@@ -354,10 +357,21 @@ func buildAlbumID3(_ context.Context, album model.Album) responses.AlbumID3 {
 	dir.Moods = album.Tags.Values(model.TagMood)
 	dir.DisplayArtist = album.AlbumArtist
 	dir.Artists = artistRefs(album.Participants[model.RoleAlbumArtist])
+	dir.ExplicitStatus = setExplicitStatus(album.ExplicitStatus)
 
 	return dir
 }
 
+func setExplicitStatus(explicitStatus string) string {
```

Comment:
IMO, this is more of a _getter_ than a _setter_. Maybe rename it to just `explicitStatus()`?